perfkitbenchmarker/cloudstack/cloudstack_virtual_machine.py

```diff
@@ -188,13 +189,14 @@ def CreateScratchDisk(self, disk_spec):
 
     for i in xrange(disk_spec.num_striped_disks):
 
-        name = 'disk-%s-%s' % (self.name, i + 1)
+        name = 'disk-%s-%d-%d' % (self.name, i + 1, self.disk_counter)
```

Comment:
It would when you create two disks. I assume that `CreateScratchDisk()` will be called for each disk. In that case the name would not be unique. Atleast that is what I see from the logs.